store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1629,22 +1629,33 @@ private boolean isTempFileExist() {
         return file.exists();
     }
 
-    private void recover(final boolean lastExitOK) {
-        long recoverCqStart = System.currentTimeMillis();
-        long maxPhyOffsetOfConsumeQueue = this.recoverConsumeQueue();
-        long recoverCqEnd = System.currentTimeMillis();
+    private boolean recover(final boolean lastExitOK, final boolean recoverConcurrently) {
```

Comment:
How about keeping the original function signature？ We can also get the brokerConfig in the recover function.